client/src/Components/Overview/Gallery.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+
+
+const Gallery = () => {
+  const [productURL, setProductUrl] = useState('');
+  const [styles, setStyles] = useState([]);
+  const [currentStyle, setCurrentStyle] = useState([]);
+
+  const getProduct = () => {
+    axios.get('/products/40348/styles')
```

Comment:
This part can be refactored to take a variable product id

Reply:
I am working on that.